templates/ts/action/.vscode/settings.json

```diff
@@ -0,0 +1,11 @@
+{
+    "debug.onTaskErrors": "abort",
+    "json.schemas": [
+        {
+            "fileMatch": [
+                "/aad.*.json"
```

Comment:
what's purpose for this aad file?

Reply:
This part is to suppress aad.manifest.json's weird intellisense. Old version of TTK has "add sso" command so all templates have this part in `settings.json`. Since current TTK doesn't have "add sso" anymore, I will remove this part.